### ISSUE TITLE
Modifications for GSD ensemble use case

### ISSUE THREAD

**Issue by jimfrimel** [NCAR/METplus#112]
Create a branch for modifications to run the NOAA/ESRL/GSD ensemble use case.

At some point this will probably meld with HWT branch based off issue #111

**Comment by jimfrimel** [comment]
An initial release of the HRRR ensemble use case for GSD is implemented.
Current review, testing and evaluation has begun by GSD

This implementation only supports the requirements from the initial
ensemble surface use case which was verification by initialization times.

KEEPing the feature_112_gsd_ensemble  branch during T and E.
feature_112_orig_ensemble_test branch was removed

The obs_error_table.txt file changed from MET version 7.1_beta6 to 8.0
Added support for this change to the branch
commit 4f395f75642a81

**Comment by jimfrimel** [comment]
We were not able to produce repeatable results since we did not have
a seed set in the random number generator.  

Added the rng random number generator to the Ensemble Stat config file
and a seed entry in order to produce repeatable output.
    
Changed the output flag to BOTH to produce individual statistical output
files. 

I was able to produce the same results when comparing MET 8.0 output from 
Jeff Beck's original scripts vs. the Calling MET from the METplus.
I only confirmed F00 and F36.  Handing over to Jeff Duda for validation of results.
